frontend/src/components/pages/TournamentsList/TournamentsList.tsx

```diff
@@ -0,0 +1,23 @@
+import CardTitleWithNavButton from "../../molecules/CardTitleWithNavButton/CardTitleWithNavButton";
+import {VerticalSpace} from "../../atoms/Shared/VerticalSpace";
+import TournamentCardsGallery from "../../organisms/TournamentsCardsGallery/TournamentsCardsGallery";
+import PlusButton from "../../atoms/Shared/PlusButton/PlusButton";
+import {styled} from "@material-ui/core";
+
+const StyledWrapper = styled("div")({
+    position: 'relative'
+})
+
+const TournamentsList = () => {
+    return (
+        <StyledWrapper>
+            <CardTitleWithNavButton title={"Turnieje"}/>
+            <VerticalSpace height={30}/>
```

Comment:
Czy VerticalSpace przyjmuje number oraz stringa? 
Ale numer nie działa. Bo nie ma tam przypisania "px". A bez "px" (czyli stringa) to nie zadziała

![image](https://user-images.githubusercontent.com/41823689/114593092-ff260100-9c8b-11eb-8422-6e55bc1e7c5f.png)

Reply:
działa, z automatu dopisuje się 'px' przy wartości number